MissedRevenueHeatMapGenerator.py

```diff
@@ -0,0 +1,67 @@
+class MissedRevenueHeatmapGenerator:
+
+    #METHODS
+    #====================================================================
+    def calculate_hours_a_branch_is_open(self, branch, day_of_the_week, business_hours_dict):
+        '''return the hours (integer) a branch is open'''
+        #TODO: Implement this
+        #return hours a branch is open
+        pass
+
+    def calculate_low_stock_level_periods(self, sku, branch, journal, danger_level=0):
```

Comment:
updated comments.
this function should return a list of tuples, which represent periods in time for which stock level was below the danger level for that SKU